.config/sample-config.json

```
@@ -43,6 +43,38 @@
             ],
             "thumbnailPath": "assets/thumbnail.png",
             "suggested": false
+        },
+        {
+            "id": "excel-add-in-chartapi-analyze-data",
+            "onboardDate": "2024-11-11",
+            "title": "Use the Excel Chart APIs to analyze data",
```

Let's lead with the goal, rather than the feature.
```suggestion
            "title": "Analyze Excel data with charts",
```